spec/07-variables.md

```diff
@@ -259,6 +281,8 @@ function f()
 }
 ```
 
+Be also aware that declaring a variable global can hide a local variable and/or a function static with the same name. See ([§§](#hidingNotice)) for an example.
```

Comment:
We have 145 closed pulls and 5 open, so I don't think we are doing that badly. Some pulls take longer because they need more fixes, or because they came in unlucky timing where all volunteers were busy with other things.